### PR TITLE
Review Java Testing article

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

File: java/pantheon_testing.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+## Running a Node with Testcontainers
 
-Testcontainers is a super useful library that allows you to fire up a Docker container programmatically within your test code.  Luckily, there are a number of Ethereum clients that have ready-made Docker containers uploaded to Dockerhub, which makes this task even easier.
+[Testcontainers](https://www.testcontainers.org/) is a useful library that allows you to fire up a Docker container programmatically within your test code, and there are a number of Ethereum clients that have ready-made Docker containers uploaded to Dockerhub, which makes this task easier.
 
-In this guide, I'll describe how to start and shutdown a [Pantheon](https://github.com/PegaSysEng/pantheon) node during your integration tests, so that you do not have to be concerned with starting a node manually or within your CI pipeline
+In this guide, I describe how to start and shutdown a [Pantheon](https://github.com/PegaSysEng/pantheon) node during your integration tests, so you don't have to start a node manually or within your CI pipeline
 
 ## Starting Pantheon
+
```
Comment (ChrisChinchilla) [inline java/pantheon_testing.md:31]: @craigwilliams84 My first stumbling block is here. The testcontainers docs are also lacking, but what import statements do I need etc? What's the architecture of the application around this class?

**Comment by ChrisChinchilla** [comment]
@craigwilliams84 I mostly just feel that this is missing a little bit of context around the code. You could argue that a developer _should_ know that anyway. if you like, share some project code with me and I'll fill in the gaps.

**Comment by ChrisChinchilla** [comment]
@craigwilliams84 You'll be pleased to hear I am done! If you're happy, merge away and publish!

**Review by craigwilliams84 — APPROVED** [review]
Looks good to me!

**Pull request merged**